pydeps/cli.py

```diff
@@ -175,7 +176,12 @@ def parse_args(argv=()):
         _args.show_dot = False
     if _args.max_bacon == 0:
         _args.max_bacon = sys.maxsize
-    if _args.keep_target_cluster or _args.min_cluster_size > 0 or _args.max_cluster_size > 0:
+    if (
+        _args.keep_target_cluster
+        or _args.min_cluster_size > 0
+        or _args.max_cluster_size > 0
+        or _args.collapse_target_cluster
+    ):
         _args.cluster = True
```

Comment:
Forcing `--cluster` happens here. I would prefer to enable `--collapse-target-cluster` without this but it turned out to be non-trivial 😞

Reply:
I think that is ok. For non-trivial projects it's usually quite "messy" outside the target cluster so this is a good default. Add a new issue about it if you feel this would be a problem.